packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -273,6 +274,16 @@ export class FunctionPluginImpl {
     return ResultFactory.Success(res);
   }
 
+  public async executeUserTask(func: Func, ctx: PluginContext): Promise<FxResult> {
+    if (func.method === CustomizedTask.addResource) {
+      TelemetryHelper.sendGeneralEvent(FunctionEvent.addResource);
+      await this.preScaffold(ctx);
```

Comment:
We'd better to check the ret value according to our design. I know it is redundant for current logic, but we need to code for principle.

Reply:
Agree. Updated.